bazel/repositories.bzl

```diff
@@ -399,6 +400,25 @@ def _com_google_absl():
         name = "abseil_optional",
         actual = "@com_google_absl//absl/types:optional",
     )
+    native.bind(
+        name = "abseil_synchronization",
+        actual = "@com_google_absl//absl/synchronization:synchronization",
+    )
+    native.bind(
+        name = "abseil_time",
```

Comment:
Can you add a `TODO` here to replace the use of bind and external_deps with just the direct Bazel path at all sites? E.g. `@com_google_absl//absl/synchronization:synchronization` should just be the in `deps` of an `envoy_cc_library`. This is now the preferred method for pure Bazel deps. OTOH, it makes sense to do this in bulk for the entire Abseil dep, which might be out of scope for this PR.

Reply:
done

Reply:
Is anyone using `abseil_time` now?